src/components/Settings/Export/Export.helpers.js

```diff
@@ -598,6 +631,7 @@ const addTileToGrid = async (
   const labelData = {
     text: label,
     alignment: 'center',
+    fontSize: exportAllBoardSize,
```

Comment:
See line 643  - 
`if (imageData.width <= 37) {
    labelData.fontSize = 7;
  } else if (imageData.width <= 40) {
    labelData.fontSize = 8;
  } else if (imageData.width <= 45) {
    labelData.fontSize = 9;
  }`

We need to manage also this!